assets/migrations/2019_09_15_000020_create_domains_table.php

```diff
@@ -18,10 +18,10 @@ public function up(): void
         Schema::create('domains', function (Blueprint $table) {
             $table->increments('id');
             $table->string('domain', 255)->unique();
-            $table->string('tenant_id');
+            $table->string('tenant_id')->nullable();
```

Comment:
Does the `nullable()` need to be here? Since you removed the `set null`

Reply:
No, it does not. Forgot to remove it. Sorry for the inconvenience.